pywebpush/tests/test_webpush.py

```diff
@@ -211,61 +209,67 @@ def test_webpush_vapid_exp(self, vapid_sign, pusher_send):
         )
         vapid_sign.assert_called_once_with(claims)
         pusher_send.assert_called_once()
-        ok_(claims['exp'] > int(time.time()))
+        assert claims['exp'] > int(time.time())
 
     @patch("requests.post")
     def test_send_bad_vapid_no_key(self, mock_post):
         mock_post.return_value.status_code = 200
 
         subscription_info = self._gen_subscription_info()
         data = "Mary had a little lamb"
-        assert_raises(WebPushException,
-                      webpush,
-                      subscription_info=subscription_info,
-                      data=data,
-                      vapid_claims={
-                          "aud": "https://example.com",
-                          "sub": "mailto:ops@example.com"
-                      }
-                      )
+        try:
```

Comment:
nit: py.test raises should work here I think

```suggestion
        with pytest.raises(WebPushException):
```

Reply:
or `self.assertRaises`